### PR TITLE
chore: SIMA application/license schema changes

### PR DESCRIPTION
**Pull request by acarnagey**
Removing optional schema fields not in the example

### CODE REVIEW

File: docs/openapi/components/schemas/common/SIMASteelImportLicense.yml [hunk 1]
```diff
@@ -16,13 +16,7 @@ properties:
       type: string
       enum:
         - SIMASteelImportLicense
-  licenseNumber:
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/SIMASteelImportLicense.yml:19]: We can't remove this, they are based on SIMA's documentation.

File: docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml [hunk 2]
```diff
@@ -24,13 +24,6 @@ properties:
     $linkedData:
       term: steelProduct
       '@id': https://w3id.org/traceability#SteelProduct
-  productCategory:
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml:27]: We can't remove this, they are based on SIMA's documentation.

**Review by nissimsan — CHANGES_REQUESTED** [review]
If these are missing example values, then the fix should be adding those values. Not removing them from the schema.

File: docs/openapi/components/schemas/common/SIMASteelImportLicense.yml (hunk 1 above)
Comment (BenjaminMoe) [reply docs/openapi/components/schemas/common/SIMASteelImportLicense.yml:19]: that means it should be in the example, right?

File: docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml (hunk 2 above)
Comment (BenjaminMoe) [reply docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml:27]: that means it should be in the example, right?

Comment (nissimsan) [reply docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml:27]: Yep!

**Comment by OR13** [comment]
74ffzc

**Comment by acarnagey** [comment]
@nissimsan should these then be required fields in the schema, should we also keep inspection observation in steel product too?

File: docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml (hunk 2 above)
Comment (acarnagey) [reply docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml:27]: @nissimsan what would the example value look like for this?

Comment (acarnagey) [reply docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml:27]: I think I figured this out went with 'Long (Carbon and Alloy)' based off of https://www.trade.gov/steel-products-hts-codes

**Comment by acarnagey** [comment]
@nissimsan took another pass at this, let me know if there should be some other ones I removed that should be required and added the example

File: docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml [hunk 3]
```diff
@@ -37,24 +37,15 @@ properties:
         - SIMASteelImportLicenseApplicationCredential
   id:
     type: string
-  name:
-    type: string
-  description:
-    type: string
   issuanceDate:
     type: string
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml:41]: This should be `date-time` too?

File: docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml [hunk 4]
```diff
@@ -37,24 +37,15 @@ properties:
...
-  expirationDate:
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml:46]: `expirationDate` can stay in as an optional field.

File: docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml [hunk 5]
```diff
@@ -37,24 +37,15 @@ properties:
...
-    type: string
+    format: date-time
   issuer:
     $ref: ../common/Organization.yml
   credentialSubject:
     $ref: ../common/SIMASteelImportLicense.yml
   proof:
     $ref: ../snippets/proof.yml
-  relatedLink:
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml:54]: Related link can stay in as an optional field.

File: docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml [hunk 6]
```diff
@@ -37,24 +37,15 @@ properties:
         - SIMASteelImportLicenseApplicationCredential
   id:
     type: string
-  name:
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml:40]: name should be in the example, but not required. "SIMA Steel Import License Application"

File: docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml [hunk 7]
```diff
@@ -37,24 +37,15 @@ properties:
...
-    type: string
-  description:
-    type: string
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/SIMASteelImportLicenseApplicationCredential.yml:43]: description should be in the example, but not required.

File: docs/openapi/components/schemas/credentials/SIMASteelImportLicenseCredential.yml [hunk 8]
```diff
@@ -37,16 +37,14 @@ properties:
         - SIMASteelImportLicenseCredential
   id:
     type: string
-  name:
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/SIMASteelImportLicenseCredential.yml:40]: name and description should be included.

**Review by BenjaminMoe — APPROVED** [review]

**Review by nissimsan — APPROVED** [review]

**Comment by nissimsan** [comment]
Solid update, @acarnagey! 

In summary: 
- Narrow the SteelProduct schema
- Adds format: date-time
- Tighten up the json schema test

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**